IDEAS/Buildings/Components/BaseClasses/RadiativeHeatTransfer/Examples/ZoneLwDistribution.mo

```diff
@@ -49,7 +49,7 @@ equation
   connect(ramp.y,preTem. T)
     annotation (Line(points={{-79,0},{-62,0}}, color={0,0,127}));
   connect(preTem.port, zonLwDist.port_a[1]) annotation (Line(points={{-40,0},{
-          -20,0},{-20,69.1667},{-10,69.1667}},
+          -20,0},{-20,69.5833},{-10,69.5833}},
```

Comment:
This is due to Dymola, only way to avoid this is in models that need changes is to apply these changes in a text editor instead of Dymola. I'll fix it for this issue manually